TicTacToeApp/Game.swift

```diff
@@ -0,0 +1,36 @@
+
+public class Game {
+
+  private var board: Board
+  private var currentPlayer: PlayerMark
+
+  public init() {
+    board = Board()
+    currentPlayer = PlayerMark.X
+  }
+
+  public func makeMove(boardSpace: Int) {
+    board = board.markSpace(boardSpace, mark: currentPlayer)
+    currentPlayer = otherPlayer()
+  }
+
+  public func getCurrentPlayer() -> PlayerMark {
```

Comment:
Yeah, turns out I don't like it as a computed property with a getter and setter - and the setter is required, since I can't modify it internally without one. I can chat with you / show you why at your convenience.